scripts/components/OwnerRepairRequestStatus.jsx

```diff
@@ -0,0 +1,121 @@
+var React = require('react');
+var moment = require('moment');
+var Api = require('../utils/Api.js');
+var MuiContextified = require('./MuiContextified.jsx');
+var mui = require('material-ui');
+var SelectField = mui.SelectField;
+var RaisedButton = mui.RaisedButton;
+var Dialog = mui.Dialog;
+
+var OwnerRepairRequestStatus = React.createClass({
+  getInitialState() {
+    return {
+      repairStatus: '',
```

Comment:
The initial state of the dropdown should be with the __current__ status selected, not no status. Its bad usability otherwise.